brave/browser/brave_permission_manager.cc

```diff
@@ -93,7 +93,7 @@ int BravePermissionManager::RequestPermissions(
 
     if (web_contents) {
       render_frame_id = render_frame_host->GetRoutingID();
-      url = web_contents->GetURL();
+      current_origin = web_contents->GetLastCommittedURL();
```

Comment:
are you sure this is the right thing to do here? I think it's probably ok, but just wondering if there are cases here where we might be looking at the pending entry?

Reply:
`current_origin` signifies the origin of the current frame where the notification will be displayed. 

In case of link navigations: 

For example: example.com hosts the html page - 

```
<a href='sms://brave-demo.com'>Test</a>. 
```

On click, the notification will be displayed on `example.com` but since the target url is not in the current frame, I don't think there is a way to fetch it from the current web_contents object.